### ISSUE TITLE
Emit `ADD_RATING` for developer replies too (or a specific one)

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#20168]
For new ratings replies, we don't emit an `ActivityLog`:

https://github.com/mozilla/addons-server/blob/d338d271d92c400887f1e3ccf6aea6488cb4d5dc/src/olympia/ratings/models.py#L290-L295

I've added a comment when I refactored the code a few years ago now, but this has always been the case. There is no good reason though, we might be interested in tracking this as well. Possibly through a different activity class to differentiate easily between regular ratings and replies.

**Comment by GregSherman** [comment]
Requesting assignment @diox. Also, I've noticed that as a non-dev user that I cannot reply to user reviews nor developer responses to my review, so the only case where a reply_to can exist is if it is from a developer? Or are there any other cases?

**Comment by diox** [comment]
Correct, the only case where we can have a reply is from developers replying to one of their users.

**Comment by ioanarusiczki** [comment]
@GregSherman @eviljeff 

I verified this in admin -dev -> /models/activity/activitylog/ 

I see the developer replies being registered 

![developer reply](https://github.com/mozilla/addons-server/assets/33448286/ada4e61a-a515-475f-a58f-56cbeac9497c)

I've also noticed what Greg wrote [here](https://github.com/mozilla/addons-server/pull/20689#issuecomment-1547972873) .

 I'll mark this verified.

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-201
